data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/RdsSourceConfig.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.rds;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import org.opensearch.dataprepper.plugins.source.rds.configuration.AwsAuthenticationConfig;
+import org.opensearch.dataprepper.plugins.source.rds.configuration.EngineType;
+import org.opensearch.dataprepper.plugins.source.rds.configuration.ExportConfig;
+
+import java.util.List;
+
+/**
+ * Configuration for RDS Source
+ */
+public class RdsSourceConfig {
+
+    /**
+     * Identifier for RDS instance or Aurora cluster
+     */
+    @JsonProperty("db_identifier")
```

Comment:
Is this the same thing that RDS calls a `DBClusterIdentifier`? If so, it could make sense to name it to `db_cluster_identifier`.

An RDS database can contain multiple "databases" within it. Each having its own tables.

Relatedly, do we need a field for "database" somewhere?

Reply:
> Is this the same thing that RDS calls a DBClusterIdentifier? If so, it could make sense to name it to db_cluster_identifier.

RDS has separate APIs for clusters (Aurora cluster or RDS multi-az cluster) and instances (RDS single-az or multi-az instances), so there is `DBClusterIdentifier` as well as `DBInstanceIdentifier`. I'm trying to combine the two with the "db_identifier" parameter. 

Taking a closer look, one can actually have the same id for a cluster and an instance. So I may need another parameter to tell whether the id refers to a cluster or an instance, like
```
  db_identifier: my-db
  cluster: true
```
or have both `db_cluster_identifier` and `db_instance_identifier` so user can fill out one of the two.

Reply:
> An RDS database can contain multiple "databases" within it. Each having its own tables.

The "database" is currently embedded in table names, for mysql, the table name will be in `database.table` format. For postgres, there will be another layer of schema, the table name will be in `database.schema.table` format.